### PR TITLE
Added tls support

### PR DESCRIPTION
**Pull request by MeirShpilraien**
Require this fix: https://github.com/RedisLabsModules/RLTest/pull/160

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -195,6 +195,15 @@ jobs:
           test_params: VALGRIND=1
           getredis_params: --valgrind
 
+  tls:
+    docker:
+      - image: redisfab/rmbuilder:6.2.5-x64-bullseye
```
Comment (chayim) [inline .circleci/config.yml:200]: Might as well use 6.2.6 if we're already here.

File: .circleci/config.yml [hunk 2]
```diff
@@ -397,6 +406,8 @@ workflows:
               platform: [hirsute, focal, bionic, xenial, centos7, rocky8, bullseye]
       - valgrind:
           <<: *on-integ-and-version-tags
+      - tls:
+          <<: *on-integ-and-version-tags
```
Comment (chayim) [inline .circleci/config.yml:410]: Any reason we don't want this on all branches?

File: pytest/run_tests_tls.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+$HERE/generate_tests_cert.sh
```
Comment (chayim) [inline pytest/run_tests_tls.sh:5]:
```
${HERE}
```

File: pytest/run_tests_tls.sh [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+HERE="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
```
Comment (chayim) [inline pytest/run_tests_tls.sh:3]: Can't your just use HERE=`${BASH_SOURCE[0]}` or do you need the full working directory?

**Review by chayim — APPROVED** [review]
Small changes. I'm approving either way as they're not critical.

File: .circleci/config.yml (hunk 2 above)
Comment (MeirShpilraien) [reply .circleci/config.yml:410]: Juts taking time and I do not think it is needed...

File: pytest/run_tests_tls.sh (hunk 4 above)
Comment (MeirShpilraien) [reply pytest/run_tests_tls.sh:3]: I prefer a full working folder.

File: .circleci/config.yml (hunk 1 above)
Comment (MeirShpilraien) [reply .circleci/config.yml]: Fixed

File: pytest/run_tests_tls.sh (hunk 3 above)
Comment (MeirShpilraien) [reply pytest/run_tests_tls.sh]: Fixed

**Pull request merged**